src/lib.rs

```diff
@@ -319,6 +319,11 @@ where
 /// Implement this trait and [`portus::Flow`](./trait.Flow.html) to define a CCP congestion control algorithm.
 /// Instances of this type implement functionality which applies to a given
 /// algorithm as a whole.
+/// 
+/// It is recommended that you also implement
```

Comment:
Instead of this language, can we just write:

> Implement this trait, `portus::CongAlgBuilder`, and `portus::Flow` to define a CCP congestion control algorithm. `portus::CongAlg` and `portus::Flow` interact with the algorithm runtime, and `CongAlgBuilder` specifies how the `impl CongAlg` should be built from given command line arguments.

I think this language makes it more likely people will implement this, even if they just want to do things in their own crate.